mod-1/bloco-05/5.1/exercicios/index.html

```diff
@@ -45,14 +45,18 @@ <h1 class="title">Exercício 5.1 - JavaEscripito </h1>
 
       changeMainText("Daqui a 2 anos vou estar trabalhando como web dev para empresas gringas, ganhando em dólares, e estudando sobre game dev.");
 
-      function changeSideColor(rgbColorString) {
-        console.log(document.getElementsByClassName('main-content')[0]);
-
-        document.getElementsByClassName('main-content')[0].style.backgroundColor = rgbColorString;
+      function changeSideColor(colorString) {
```

Comment:
Boa noite Inácio, já que as duas funções estão fazendo a mesma coisa (alterando o background color de uma classe), não seria interessante alterar sua função para ela mudar a backgroundColor de acordo com a class? (ambos passados por argumentos para a função) (className, colorString)

Reply:
É uma ótima ideia mesmo!